### PR TITLE
REPO-3682 : Renditions: Deployment includes queue by default

### PR DESCRIPTION
**Pull request by cristianturlica**
- change activemq image to alfresco/alfresco-activemq
  - change liveliness and readiness probes
  - added properties for brokerName, adminLogin and adminPassword

### CODE REVIEW

**Comment by killerboot** [comment]
It would be nice if we can release the chart to stable and possibly include into an infrastructure chart. So that ACS can be based only on released to stable charts.

**Review by nicdoye — APPROVED** [review]
LGTM! 

_(Note that we require two approvals before a merge)._

lgtm3

**Review by rgauss — APPROVED** [review]

**Pull request merged**
